test/integration/kubernetes_deploy_test.rb

```diff
@@ -306,32 +308,40 @@ def test_invalid_k8s_spec_that_is_valid_yaml_but_has_no_template_path_in_error_p
   end
 
   def test_bad_container_image_on_unmanaged_pod_halts_and_fails_deploy
-    result = deploy_fixtures("hello-cloud") do |fixtures|
-      pod = fixtures["unmanaged-pod.yml.erb"]["Pod"].first
+    result = deploy_fixtures(
+      "hello-cloud",
+      subset: [
+        "unmanaged-pod-1.yml.erb",
+        "configmap-data.yml",
+        "redis.yml",
+        "web.yml.erb",
```

Comment:
Better to create a new fixture directory for the multiple-bare-pods case and leave `hello-cloud` alone?

Reply:
I don't have strong feelings about this.